webapp/src/ui/components/courses-list.js

```diff
@@ -22,7 +22,7 @@ const columns = [
 const CoursesList = ({data}) => {
```

Comment:
Not sure if this is a git rendering bug or not but all of these are 4 spaces on github's UI and then 2 in the code. This could also be a fault of my RC files automatically taking control. Our existing codebase uses 2 spaces and I see no reason to change from 2 to 4

Reply:
They're tabs, not spaces - not sure why they're rendering as 4 spaces.
I'll change the linter to use spaces instead.

Reply:
Fixed by 762accb. Also updated ESLint config for editor integration.